module/VuFind/tests/unit-tests/src/VuFindTest/Record/RouterTest.php

```diff
@@ -95,10 +95,10 @@ public function testTabRoutingWithString()
     public function testCollectionSpecialCaseWithString()
     {
         $driver = $this->getDriver();
-        $driver->expects($this->once())->method('tryMethod')->with($this->equalTo('isCollection'))->will($this->returnValue(true));
+        $driver->expects($this->never())->method('tryMethod')->with($this->equalTo('isCollection'))->will($this->returnValue(true));
```

Comment:
Probably should remove the `->will` clause, since it doesn't make sense to define behavior for something that should never happen.

Reply:
Right.